docs/conf.py

```diff
@@ -18,37 +18,37 @@
 from pathlib import Path
 
 sys.path.insert(0, os.path.abspath(".."))
-repo_dir = Path(__file__).parent.parent
+REPO_DIR = Path(__file__).parent.parent
 
 # -- Project information -----------------------------------------------------
 
-project = "Astronomer Providers"
-author = "Astronomer Inc."
+PROJECT = "Astronomer Providers"
```

Comment:
We can't do the others though as they are using by Sphinx.

We only can do that are used internally, like `REPO_DIR`. Feel free to make it internal if you want to by adding `_` i.e. `_REPO_DIR`